.github/workflows/template-build.yml

```
@@ -0,0 +1,87 @@
+on:
+  workflow_call:
+    inputs:
+      release-tag-name:
+        required: true
+        type: string
+      push:
+        required: true
+        type: boolean
+
+jobs:
+  dapper-build:
+    runs-on: ubuntu-latest
+    container:
+      image: rancher/dapper:v0.5.8
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - linux/amd64
+          - linux/arm64
+    steps:
+      # Git is not in Dapper container image. Add it manually for dirty check.
+      - name: Add Git
+        run: apk add -U git
+
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Get Arch
+        id: get-arch
+        run: |
+          arch=$(echo ${{ matrix.platform }} | cut -c 7-)
+          echo "ARCH=$arch" >> "$GITHUB_OUTPUT"
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Run dapper
+        run: dapper ci
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v3
+        if: ${{ inputs.push == true }}
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Docker Publish Master
```

```suggestion
      - name: Docker Publish
```

it's used as a templated workflow, suggest not naming it with master.